docs/src/main/sphinx/connector/snowflake.md

```diff
@@ -49,31 +49,54 @@ multiple instances of the Snowflake connector.
 
 ## Type mapping
 
-Trino supports the following Snowflake data types:
-
-| Snowflake Type | Trino Type     |
-| -------------- | -------------- |
-| `boolean`      | `boolean`      |
-| `tinyint`      | `bigint`       |
-| `smallint`     | `bigint`       |
-| `byteint`      | `bigint`       |
-| `int`          | `bigint`       |
-| `integer`      | `bigint`       |
-| `bigint`       | `bigint`       |
-| `float`        | `real`         |
-| `real`         | `real`         |
-| `double`       | `double`       |
-| `decimal`      | `decimal(P,S)` |
-| `varchar(n)`   | `varchar(n)`   |
-| `char(n)`      | `varchar(n)`   |
-| `binary(n)`    | `varbinary`    |
-| `varbinary`    | `varbinary`    |
-| `date`         | `date`         |
-| `time`         | `time`         |
-| `timestampntz` | `timestamp`    |
-
 Complete list of [Snowflake data types](https://docs.snowflake.com/en/sql-reference/intro-summary-data-types.html).
 
+### When Trino reads a Snowflake table
```

Comment:
@mosabua Do you want to review this docs change?

Reply:
Sure ..